src/DistributedLock.Azure/packages.lock.json

```diff
@@ -18,15 +18,6 @@
         "resolved": "3.3.4",
         "contentHash": "kNLTfXtXUWDHVt5iaPkkiPuyHYlMgLI6SOFT4w88bfeI2vqSeGgHunFkdvlaCM8RDfcY0t2+jnesQtidRJJ/DA=="
       },
-      "Microsoft.NETFramework.ReferenceAssemblies": {
```

Comment:
This was added in https://github.com/madelson/DistributedLock/commit/78f3386b403e122565d7396e9b3815de03aea367, but I'm not sure why this was a direct dependency. Maybe your VS is set up in a certain way?

Reply:
Yeah not sure. This came in when I built. I'm OK with some thrash here.